contributors/guide/get-started.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Getting Started"
+weight: 3
+slug: "getting-started" 
```

Comment:
The slug for this needs to match the filename -- in this case, I'd rename the file to be `getting-started.md` (and update the link in the README)
To be honest, we can probably remove the slug field for almost all the docs in the contributor guide. :x They're only needed under a few specific use-cases.

Reply:
No issue I will change the file name